modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/StructuredContentResourceImpl.java

```diff
@@ -1247,18 +1189,16 @@ private void _populateContentFieldValuesMap(
 			ContentFieldValue contentFieldValue =
 				contentField.getContentFieldValue();
 
-			if ((contentFieldValue != null) &&
-				(_getValue(contentFieldValue) != null)) {
-
-				List<ContentFieldValue> contentFieldValues =
-					contentFieldValuesMap.computeIfAbsent(
-						contentField.getName(), key -> new ArrayList<>());
-
-				contentFieldValues.add(contentFieldValue);
+			if (contentFieldValue != null) {
+				contentFieldsMap.computeIfAbsent(
+					contentField.getFieldReference(), key -> new ArrayList<>()
+				).add(
+					contentField
+				);
```

Comment:
In our code method chaining is not recommended. Please separate the `add()` call.